cheroot/ssl/builtin.py

```diff
@@ -172,22 +235,155 @@ def get_environ(self, sock):
             'HTTPS': 'on',
             'SSL_PROTOCOL': cipher[1],
             'SSL_CIPHER': cipher[0],
-            # SSL_VERSION_INTERFACE     string  The mod_ssl program version
-            # SSL_VERSION_LIBRARY   string  The OpenSSL program version
+            'SSL_CIPHER_EXPORT': '',
+            'SSL_CIPHER_USEKEYSIZE': cipher[2],
+            'SSL_VERSION_INTERFACE': 'Python %s' % sys.version,
+            'SSL_VERSION_LIBRARY': ssl.OPENSSL_VERSION,
+            'SSL_CLIENT_VERIFY': 'NONE',
         }
 
+        # Python 3.3+
+        if hasattr(sock, 'compression'):
+            compression = sock.compression()
+            if compression is not None:
+                ssl_environ['SSL_COMPRESS_METHOD'] = compression
+
+        # Python 3.6+
+        if hasattr(sock, 'session'):
+            ssl_environ['SSL_SESSION_ID'] = sock.session.id.hex()
+        if hasattr(sock.context, 'get_ciphers'):
+            for cip in sock.context.get_ciphers():
+                if cip['name'] == cipher[0] and cip['protocol'] == cipher[1]:
```

Comment:
it may read easier as
```python
target_cipher = cipher[:2]
```
```suggestion
                if target_cipher == (cip['name'], cip['protocol']):
```

Reply:
done